### DISCUSSION TITLE
Searching a user results in empty json

### DISCUSSION THREAD

**Discussion by felixriehm** [mastodon/mastodon#21314] [Q&A] [answered]
I have a self hosted instance and everything works fine so far, however users from other instances can not find me. If they search for 

`https://mydomain/@myusername`

they will not find anything. The browser sends this GET request

`https://theirdomain/api/v2/search?q=https://mydomain/@myusername&resolve=true&limit=5`

which results in `{"accounts":[],"statuses":[],"hashtags":[]}`, status 200.

However, myusername exists on my instance and also has some toots.

I don't get any error or warning messages with any services (mastodon-web, mastodon-services or mastodon-sidekiq). What could cause this problem? Or how can i further investigate?

**Comment by nightpool** [comment]
It's impossible for us to help you with this if you don't share your actual domain so that we can make the request ourselves and see the response.

**Reply by felixriehm** [reply]
@nightpool sry, you can test it here https://social.mpdl.mpg.de. For instance, if you search for https://social.mpdl.mpg.de/@riehm nothing will be found.

**Comment by felixriehm** [comment, marked as answer]
For everyone finding this via google:
I have solved this with the fix discussed here -> https://github.com/mastodon/mastodon/issues/6449

It was due to an incomplete ssl certificate. For some reason you only get an error if you search for "@username@domain", when searching with an url (e.g. displayed in a modal window when interacting with a remote post) you do not get that error message.
